Homework_2/Task_2.py

```diff
@@ -0,0 +1,6 @@
+#2 –Напишите программу, которая принимает на вход число N и выдает набор произведений чисел от 1 до N.
+N= int(input("Введите N "))
+result=[1]
+for i in range (1,N+1):
+    result.append(i*result[i-1])
```

Comment:
```suggestion
for i in range (2, N + 1):
    result.append(i * result[-1])
```
Так ведь чуточку лучше, не правда ли?